caluma/caluma_core/models.py

```diff
@@ -109,18 +108,4 @@ class Meta:
 
 
 class ChoicesCharField(models.CharField):
-    """
-    Choices char field type with specific form field.
-
-    Graphene Django Filter converter uses formfield for conversion.
-    To support enum choices for arguments we need to set a label
-    which is unique across schema.
-    """
-
-    def formfield(self, **kwargs):
-        # Postfixing newly created filter with Argument to avoid conflicts
-        # with query nodes
-        meta = self.model._meta
-        name = to_camel_case(f"{meta.object_name}_{self.name}_argument")
-
-        return super().formfield(label=name, **kwargs)
+    pass
```

Comment:
Do we even need this anymore?

Reply:
Yes, for backwards compatiblity in the migrations. I added a comment and removed all implementation in favor of `models.CharField`